Sources/Brave/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3168,8 +3182,10 @@ extension BrowserViewController {
   }
 
   public func handleNavigationPath(path: NavigationPath) {
-    // Remove Default Browser Callout if an external url is triggered
+    // Remove Default Browser Callout - Do not show scheduled notification
+    // in case an external url is triggered
     Preferences.General.defaultBrowserCalloutDismissed.value = true
+    Preferences.DefaultBrowserIntro.defaultBrowserNotificationShouldBeShown.value = false
```

Comment:
Shouldn't this be set based on the path passed in? (must be `url` and the URL must be `http`/`https`)

Reply:
You are right, good catch and it should be included to other value also. Fix at https://github.com/brave/brave-ios/pull/8067/commits/5a0f5d2f3b5b421b12cf57a366178420d5bc993a